packages/react-day-picker/src/hooks/useControlledValue/useControlledValue.ts

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+export type DispatchStateAction<T> = React.Dispatch<React.SetStateAction<T>>;
+
+/**
+ * Helper hook for using controlled/uncontrolled values from a component props.
+ *
+ * When a component is controlled, pass the controller value and use the
+ * returned setter to update it.
```

Comment:
The tests look good, but this comment doesn’t match my expectation. The setter does nothing in the controlled state.

Reply:
Thanks! I've updated it in https://github.com/gpbl/react-day-picker/pull/1283/commits/c5b562c688f3fe3e02bfdae6010110d81e0952ee